src/core/structures/PDFAcroForm.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  PDFArray,
+  PDFBool,
+  PDFContext,
+  PDFDict,
+  PDFName,
+  PDFNumber,
+  PDFObject,
+  PDFString,
+} from 'src/core';
+
+class PDFAcroForm {
+  static fromDict(dict: PDFDict, context: PDFContext): PDFAcroForm {
+    if (!dict.has(PDFName.of('Fields'))) {
+      dict.set(PDFName.of('Fields'), context.obj([]));
+    }
+    return new PDFAcroForm(dict, context);
+  }
```

Comment:
It's not necessary to pass in a `PDFContext` here. Instead, we can simply use `dict.context`. I think this is preferable, actually, as it simplifies the API a bit and removes the risk of passing mismatched contexts.

For example:
```js
static fromDict(dict: PDFDict): PDFAcroForm {
  if (!dict.has(PDFName.of('Fields'))) {
    dict.set(PDFName.of('Fields'), dict.context.obj([]));
  }
  return new PDFAcroForm(dict);
}
```